src/types.ts

```diff
@@ -63,6 +63,8 @@ export interface Column<TRow, TSummaryRow = unknown> {
     readonly displayCellContent?: Maybe<boolean>;
     /** @default true */
     readonly commitOnOutsideClick?: Maybe<boolean>;
+    /** @default true */
+    readonly discardOnRowChange?: Maybe<boolean>;
```

Comment:
```suggestion
    readonly discardOnOutsideRowChange?: Maybe<boolean>;
```

or 
```suggestion
    readonly discardOnExternalRowChange?: Maybe<boolean>;
```
wdyt?

Reply:
I like "external" better.
WDYT about
```suggestion
    readonly closeOnExternalRowChange?: Maybe<boolean>;
```